app/assets/stylesheets/alchemy/shoelace.scss

```
@@ -347,6 +347,7 @@ sl-tooltip {
 sl-dialog {
   &::part(panel) {
     background-color: var(--color-grey_light);
+    min-height: var(--height);
```

Should we prefix the variable with `dialog`?

```suggestion
    min-height: var(--dialog-min-height);
```